### ISSUE TITLE
Performance issues

### ISSUE THREAD

**Issue by steph-dmts** [CommandLineHeroes/clh-bash#75]
The game uses all cores of my i5-5257U cpu at 100% and is impossible to play. I run linux so I know there's no hardware acceleration supported on browsers without going through some dirty hacks but I'm pretty sure this could be an issue for a lot of people.

**Comment by ghost** [comment]
Yeah i have the same issue and am running core i7 and 8gb ram but still very slow

**Comment by Jared-Sprague** [comment]
Yeah the game is not optimized right now.  So it requires a decent GPU to get good frame rates.  I would welcome any performance optimization PRs!

**Comment by Jared-Sprague** [comment]
Another Idea is we could make a 2D version of the game,  The game logic is decoupled from the presentation and the performance issues are all due high-res 3D objects in WebGL.  A 2D version of the game would perform well on any hardware even a pi.

**Comment by ghost** [comment]
maybe we can optimize the game interface to be just a basic terminal without 3d objects

**Comment by Jared-Sprague** [comment]
Yep! That's basically what I ment by 2D version of the game.
